src/components/HomePage/Modals/MovieModal.tsx

```diff
@@ -5,25 +5,25 @@ import { ButtonType } from '../../../model/enums/ButtonType';
 import { AppButton, Heading } from '../../shared';
 import './Modal.scss';
 import { MovieForm } from '../Forms/MovieForm';
-import { MovieInfo } from '../../../types';
+import { MovieInfoDetails } from '../../../types';
 
 export const MovieModal: FC<{
   newMovie: boolean,
-  movieInfo?: MovieInfo, onClickCallBack?: Function
+  movieInfo?: MovieInfoDetails, onClickCallBack?: Function
 }> = ({ newMovie,
   movieInfo, onClickCallBack }) => {
-  const [isOpen, toggleModal] = useToggle(false);
-  const onClick = useCallback(() => {
-    if (onClickCallBack) {
-      onClickCallBack();
-    }
-    toggleModal();
-  }, [onClickCallBack, toggleModal]);
+    const [isOpen, toggleModal] = useToggle(false);
+    const onClick = useCallback(() => {
+      if (onClickCallBack) {
+        onClickCallBack();
+      }
+      toggleModal();
+    }, [onClickCallBack, toggleModal]);
```

Comment:
we do not need to pass functions as dependencies

Reply:
fixed in commit e9860952613603eb2cf3095f28c180d207b9668c